src/bridge/lib/magic_numbers.py

```diff
@@ -38,6 +38,9 @@
 MAXIMUM_RSA_KEY_SIZE = 4096
 ONE_GIGABYTE_MB = 1024
 ONE_GIGAHERTZ = 1024
+ONE_MONTH = 60 * 60 * 24 * 30
```

Comment:
Maybe rename this to `ONE_MONTH_SECONDS` to be clear what the underlying unit of time is